src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -210,12 +211,16 @@ public partial class ComAwareEventInfo : System.Reflection.EventInfo
         public ComAwareEventInfo(System.Type type, string eventName) { }
         public override System.Reflection.EventAttributes Attributes { get { throw null; } }
         public override System.Type DeclaringType { get { throw null; } }
+        public override int MetadataToken { get { throw null; } }
+        public override System.Reflection.Module Module { get { throw null; } }
         public override string Name { get { throw null; } }
         public override System.Type ReflectedType { get { throw null; } }
         public override void AddEventHandler(object target, System.Delegate handler) { }
         public override System.Reflection.MethodInfo GetAddMethod(bool nonPublic) { throw null; }
         public override object[] GetCustomAttributes(bool inherit) { throw null; }
         public override object[] GetCustomAttributes(System.Type attributeType, bool inherit) { throw null; }
+        public override System.Collections.Generic.IList<System.Reflection.CustomAttributeData> GetCustomAttributesData() { throw null; }
```

Comment:
Because these are override, they were already available to intellisense/compiler because they existed on the base, right? Nothing new.

Reply:
Correct.

Reply:
Right, but it does affect C# compiler code generation in the (probably super rare case) where someone was deriving from these types and calling base.MetadataToken/Module/GetCustomAttributesData.  Without the overrides in the ref assembly, such calls would likely end up skipping these overrides and calling to the relevant implementations on EventInfo directly.

Reply:
But we don’t want to skip them right? Are they are actually overridden in the implementation, no?

Reply:
Correct, it was a bug that they were missing from the ref if they're overridden in the implementation and the class can be derived from.

Reply:
Ok. Yes the class is not sealed and they're in the implementation, that is why GenAPI picked them up and added them into the ref.